docs/rules/customization.md

```diff
@@ -86,5 +86,5 @@ TBD
 
 
 #### More
-* See [our rule schema definition](SCHEMA.md). 
+* See [our rule schema definition](https://github.com/mozilla/eslint-plugin-no-unsanitized/blob/main/SCHEMA.md). 
```

Comment:
Nit, This change doesn't seem to be relevant to this PR not necessary.

Reply:
I think that was added by some stupid tool. Removing.

Reply:
That's not my work, (git blames it on Alvaro Junqueira). Will remove anyway.

Reply:
On second thought, I'll keep it.